examples/tv-casting-app/android/App/app/src/main/java/com/matter/casting/DiscoveryExampleFragment.java

```diff
@@ -45,7 +45,7 @@ public class DiscoveryExampleFragment extends Fragment {
   private static final Long DISCOVERY_TARGET_DEVICE_TYPE = 35L;
   private static final int DISCOVERY_RUNTIME_SEC = 15;
   private TextView matterDiscoveryMessageTextView;
-  private TextView matterDiscoveryErrorMessageTextView;
+  public static TextView matterDiscoveryErrorMessageTextView;
```

Comment:
Since we're updating the error message to "The selected Casting Player does not support Commissioner-Generated passcode commissioning" when the 3P developer long clicks on a CastingPlayer that doesn't support Commissioner-Generated passcode commissioning. This happens in the CastingPlayerArrayAdapter class so this field needs to be public.
It does not need to be static.